### ISSUE TITLE
i_CTRL-N and i_CTRL-P  no longer work without CTRL-X in new text files

### ISSUE THREAD

**Issue by drjaska** [nvim-lua/kickstart.nvim#197]
**Describe the bug**
When using the kickstart template `i_CTRL-N` / `i_CTRL-P`  no longer work as expected without CTRL-X mode. They are supposed to complete current keyword (most commonly with previous/next occurrence which matches current prompt. `:help ins-completion` says that they should be usable outside of CTRL-X mode but they seem to do nothing for me.

`i_CTRL-X_CTRL-N` / `i_CTRL-X_CTRL-P` work as before.

`i_CTRL-N` / `i_CTRL-P`  do not seem to do anything now.

**To Reproduce**
1. Use up to date init.lua (I got this bug when upgrading from 2023-01-15 to latest)
2. Create a new text file
3. Enter insert mode
4. Type a word or few
5. Type part of a word which occurs at least once in the file
6. Press Control and N/P to autocomplete based on Next/Previous mention
7. There is no autocompletion and nothing visible happens..?

**Desktop (please complete the following information):**
- OS: Debian Testing (X11)
- Terminal: kitty / gnome-terminal

** Neovim Version **
 - Output of running `:version` from inside of neovim:

```
NVIM v0.8.3
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by runner@fv-az183-550

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/share/nvim"
```

**Comment by bubylou** [comment]
I'm having the same issue. I also notice nvim-cmp not working either with the CTRL+SPACE but I do get the autocomplete prompts and can select them with tab instead. I tried deleting my existing nvim configs then redownload from this repo. This happens on nvim version 8.0, 8.3, and 9.0. I'll try rolling back commits to see if I can find when exactly this started.

**Comment by rmerry** [comment]
The PR fix referenced above by @victorwallsten appears to have been closed and not merged; just wondering why.

**Comment by drjaska** [comment]
On a very quick test that PR seems to fully fix the issue without any noticeable downsides, tyvm @victorwallsten. I'll continue to use it and report back if there are further issues.

**Comment by feoh** [comment]
If you look at the PR, it says:

[victorwallsten](https://github.com/victorwallsten) closed this by deleting the head repository [3 weeks ago](https://github.com/nvim-lua/kickstart.nvim/pull/212#event-8907132698)

I haven't tested any of this yet, but perhaps @victorwallsten could comment?

**Comment by feoh** [comment]
Folks, I tried integrating this change to my personal config and I'm unable to test or repro because LuaSnip seems to be commandeering the completion.

I don't really have strong feelings about completions in text mode so if someone wants to give me concrete instructions as to how to proceed given my [current config](https://github.com/feoh/git_dotfiles/blob/master/.config/nvim/init.lua) I'm happy to test this further.

**Comment by victorwallsten** [comment]
> If you look at the PR, it says:
> 
> [victorwallsten](https://github.com/victorwallsten) closed this by deleting the head repository [3 weeks ago](https://github.com/nvim-lua/kickstart.nvim/pull/212#event-8907132698)
> 
> I haven't tested any of this yet, but perhaps @victorwallsten could comment?

I deleted my fork without thinking about my having an open pull request from there - a foolish mistake.

At the time I submitted the pull request, I was able to reproduce the behaviour by following the steps outlined by @drjaska so I guess the best way to test it is to use an unaltered kickstart.nvim config when trying to reproduce the behaviour. I'll have some spare time tomorrow - I'll try reproducing it again, introduce the same changes and submit another pull request if it fixes the problem.

**Comment by feoh** [comment]
If that's the case and it was unintentional, could you please re-open the PR? I don't have the power to do that for you.

**Comment by victorwallsten** [comment]
Unfortunately I can't reopen it since the repo was deleted, but I created a new fork and will submit a pull request shortly. Sorry for the mess.

I confirmed the behaviour a few minutes ago with an up-to-date, unaltered kickstart.nvim config in a fresh VM running Fedora 38 Silverblue. I created and entered a default toolbox, installed neovim (v0.9.0) via dnf along with gcc and g++, opened nvim and waited until the plugins and such had been installed. Then I exited nvim and ran 'nvim foo', typed 'a word or few wo<C-p>' and no completion was triggered. Same for <C-n>. After adding the nvim-cmp mappings the problem was fixed.
